front/app/components/admin/PostManager/components/PostTable/IdeaRow.tsx

```diff
@@ -65,128 +66,112 @@ type Props = InputProps & {
-            return (
-              <TitleLink
-                className="e2e-idea-manager-idea-title"
-                onClick={onClick}
-              >
-                <T value={idea.attributes.title_multiloc} />
-              </TitleLink>
-            );
-          },
-        },
-        {
-          name: 'published_on',
-          Component: ({ idea }) => {
-            return <FormattedRelative value={idea.attributes.published_at} />;
-          },
-        },
-        {
-          name: 'up',
-          cellProps: { singleLine: true },
-          Component: ({ idea }) => {
-            return (
-              <>
-                <Icon name="thumbs up" />
-                {idea.attributes.upvotes_count}
-              </>
-            );
-          },
-        },
-        {
-          name: 'down',
-          cellProps: { singleLine: true },
-          Component: ({ idea }: IdeaCellComponentProps) => {
-            return (
-              <>
-                <Icon name="thumbs down" />
-                {idea.attributes.downvotes_count}
-              </>
-            );
-          },
-        },
+const IdeaRow = ({
+  connectDragSource,
+  onClickCheckbox,
+  onClickTitle,
+  className,
+  activeFilterMenu,
+  phases,
+  statuses,
+  idea,
+  selection,
+}: Props & InjectedIntlProps & InjectedLocalized) => {
+  const [cells, setCells] = useState<
+    CellConfiguration<IdeaCellComponentProps>[]
+  >([
+    {
+      name: 'selection',
+      cellProps: { collapsing: true },
+      onChange: onClickCheckbox,
+      Component: ({
+        selection,
+        idea,
+        onChange,
+      }: Override<
+        IdeaCellComponentProps,
         {
-          name: 'picks',
-          cellProps: { singleLine: true },
-          featureFlag: 'participatory_budgeting',
-          Component: ({ idea }: IdeaCellComponentProps) => {
-            return <>{idea.attributes.baskets_count}</>;
-          },
-        },
-      ],
-    };
-  }
-
-  onUpdateIdeaPhases = (selectedPhases) => {
-    updateIdea(this.props.idea.id, {
-      phase_ids: selectedPhases,
-    });
-  };
-
-  onUpdateIdeaTopics = (selectedTopics) => {
-    updateIdea(this.props.idea.id, {
-      topic_ids: selectedTopics,
-    });
+          onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+        }
+      >) => {
+        return (
+          <Checkbox
+            checked={!!selection.has(idea.id)}
+            onChange={onChange}
+            size="21px"
+          />
+        );
+      },
+    },
+    {
+      name: 'title',
+      onClick: onClickTitle,
+      Component: ({ idea, onClick }) => {
+        return (
+          <TitleLink className="e2e-idea-manager-idea-title" onClick={onClick}>
+            <T value={idea.attributes.title_multiloc} />
+          </TitleLink>
+        );
+      },
+    },
+    {
+      name: 'published_on',
+      Component: ({ idea }) => {
+        return <>{timeAgo(Date.parse(idea.attributes.created_at), 'en')}</>;
```

Comment:
Need to replace with locale now that I can use the useLocale hook - thanks for catching this.

Reply:
Btw, before you could have used GetLocale. :-)